services/api/src/routes/auth.js

```diff
@@ -163,15 +163,34 @@ router
       ctx.status = 204;
     }
   )
-  .post('/logout', authenticate({ type: 'user' }), fetchUser, async (ctx) => {
-    const user = ctx.state.authUser;
-    await user.updateOne({
-      $unset: {
-        authTokenId: true,
-      },
-    });
-    ctx.status = 204;
-  })
+  .post(
+    '/logout',
+    validateBody({
+      all: Joi.boolean(),
```

Comment:
I like this...assuming it will be linked to something like "Log out of all devices" like spotify etc has?

Reply:
I will add to to settings